config/plop/plopfile.mjs

```diff
@@ -149,13 +158,20 @@ export default function (plop) {
       {
         type: 'input',
         name: 'route',
-        message: 'Page route (e.g. [locale]/search/ )?',
+        message: 'Page route (e.g. `[locale]/search/`)?',
       },
       {
         type: 'input',
         name: 'name',
         message: 'Model name in DatoCMS? (leave empty if none)',
       },
+      {
+        when: (data) => !!data.name,
+        type: 'confirm',
+        name: 'separateRouting',
+        message: (data) => `Create separate file in routing for \`${plop.getHelper('pascalCase')(data.name)}\`?`,
```

Comment:
At the moment file routing is in `@lib/routing/index.ts`,  choosing`y` will create a file akin to `@lib/routing/page.ts`.